<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana_reserva</class>
 <widget class="QWidget" name="Ventana_reserva">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cedula Cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entry_cedula_cliente">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="entry_nombre_cliente">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>401</x>
     <y>54</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre Cliente</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_pol">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>113</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Polideportivos</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_espacios">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>151</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Espacios Deportivos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tipo_espacio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_horario">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_estado">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_precio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Inicio reservación</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>255</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha fin reservación</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_inicio">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>215</y>
     <width>194</width>
     <height>23</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yy/M/d HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_fin">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>255</y>
     <width>194</width>
     <height>23</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yy/M/d HH:mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reservar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>328</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>328</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo pago</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_tipo_pago">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>336</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Efectivo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tarjeta Credito</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A plazos</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_precio_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tipo_espacio_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo Espacio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estado_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_horario_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Horario:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
